__tests__/scorecard.ts

```diff
@@ -0,0 +1,234 @@
+import {Scorecard} from "../src/scorecard"
+import {Frame, IFrame} from "../src/frame";
+import {setFrame} from "./testHelpers";
+
+
+
+describe(`Scorecard`, () => {
+    let scorecard: Scorecard;
+    beforeEach(() => {
+        scorecard = new Scorecard();
+    })
+    let ss = 100;
+
+    describe(`ss${ss}: addFrame()`, () => {
+        let i = 10;
+        it(`ss${ss}.T${i} adding a frame`, () => {
+            const rolls = [3, 3]
+            const frame: IFrame = {
```

Comment:
Can move some of this boilerplate to setup and alter the `rolls` property on a per-test level